src/ab-test-preview.js

```diff
@@ -0,0 +1,57 @@
+import { getLetter } from '../src/utils';
+
+document.querySelectorAll( '.ab-test-xb-preview' ).forEach( xb => {
+
+	const xbPostId = xb.getAttribute( 'post-id' );
```

Comment:
When changing this attribute to be a `data` attribute will need to change how the value is read:
```suggestion
	const xbPostId = xb.dataset.postId;
```

Note that data attributes are automatically camel cased when reading from the `dataset` property, so `post-id` becomes `postId` etc...